File: Task4/hospitality-hub/src/app/api-proxy/models/user-response-dto.ts
/* tslint:disable */
/* eslint-disable */
export interface UserResponseDto {
  email?: string | null;
  emailConfirmed?: boolean;
  firstName?: string | null;
  id?: number;
  lastName?: string | null;
  phoneNumber?: string | null;
  roles?: Array<string> | null;
  twoFactorEnabled?: boolean;
}

File: Task2/apz-pzpi-21-3-merkulov-arsenii-task2/src/HospitalityHub.Core/DTOs/Account/UserResponseDto.cs
namespace HospitalityHub.Core.DTOs.Account;

public record UserResponseDto
{
    public int Id { get; set; }
    public string Email { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public string PhoneNumber { get; set; }
    public bool EmailConfirmed { get; set; }
    public bool TwoFactorEnabled { get; set; }
    public List<string> Roles { get; set; }
}
